modules/services/servers/src/main/java/au/com/shiftyjelly/pocketcasts/servers/di/InterceptorModule.kt

```diff
@@ -215,9 +248,12 @@ object InterceptorModule {
 
     @Provides
     @Player
-    fun providePlayerInterceptors(): List<OkHttpInterceptor> {
+    fun providePlayerInterceptors(
+        @I18nInterceptor i18nInterceptor: Interceptor,
+    ): List<OkHttpInterceptor> {
         return buildList {
             add(publicUserAgentInterceptor.toClientInterceptor())
+            add(i18nInterceptor.toClientInterceptor())
```

Comment:
I noticed that the headers are being sent to third parties when an episode is played. For privacy reasons, we should limit these requests to our own servers only.